Update documentation for examples + consolidate filenames (#1099)
SAP/luigi: docs/application-setup.md

```
<!-- accordion:end -->

> 3. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which can be used to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.

Reviewer: ```suggestion
3. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which you can use to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.
```
I'd use the active voice here :)

```
<!-- accordion:end -->

> 3. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which can be used to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.

Reviewer: ```suggestion
3. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which you can use to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.
```
Same as above :)

```
<!-- accordion:end -->

> 3. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which can be used to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.

Reviewer: ```suggestion
3. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which you can use to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.
```
Same as above :)

```
<!-- accordion:end -->

> 3. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which can be used to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.

Reviewer: ```suggestion
3. Open the directory where Luigi is installed. Search for the `luigi-config.js` file you can use to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.
```
Same as above :)

```
<!-- accordion:end -->

> 2. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which can be used to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.

Reviewer: ```suggestion
2. Open the directory where Luigi is installed. Search for the `luigi-config.js` file which you can use to configure Luigi [navigation](navigation-configuration.md), [authorization](authorization-configuration.md), [general settings](general-setting.md) and more.
```
Same as above :)
